src/logging.h

```diff
@@ -24,6 +25,8 @@ static const bool DEFAULT_LOGTIMESTAMPS = true;
 static const bool DEFAULT_LOGTHREADNAMES = false;
 static const bool DEFAULT_LOGSOURCELOCATIONS = false;
 extern const char * const DEFAULT_DEBUGLOGFILE;
+static const int DEFAULT_DEBUGLOGROTATEKEEP = 10;
+static const int DEFAULT_DEBUGLOGMB = 1;
```

Comment:
To match current behavior, wouldn't you want `DEFAULT_DEBUGLOGMB` to be 10 and `DEFAULT_LOGROTATEKEEP` to be 1?

Reply:
Thanks, @narula, that's still a good question. The idea of those defaults is that between 10 MB and 11 MB of log messages would be retained (just after a log rotation, 10 MB because there'd be 10 rotated files at 1 MB each plus an empty `debug.log`; just before log rotation, 10 MB for the same reason plus a 1 MB `debug.log`. This matches the current disk usage if `bitcoind` is restarted often (if not, then the `debug.log` size can increase without limit).

If the defaults were `DEFAULT_DEBUGLOGMB=10` and `DEFAULT_LOGROTATEKEEP=1` then the logging storage would vary from 10 MB (just after rotation) to 20 MB (just before rotation).

One could argue that the defaults could be quite a bit larger because the current default (10 MB minimum) was chosen when disks were smaller than they are typically today. But there could be that _one_ user whose disk is almost full, and upgrading to a release with this PR would soon run their disk out to space if the defaults were larger. (You might say that a node's disk space requirement  grows indefinitely anyway as more blocks are added to the blockchain, but that's not necessarily so because nodes can be configured with block pruning enabled.)